201-rbac-managedidentity-maps/azuredeploy.parameters.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "mapsAccountName": {
+      "value": "GEN-UNIQUE[10]"
```

Comment:
good to know, reason I did this is because the testing script:
`.\Deploy-AzureResourceGroup.ps1 -ArtifactStagingDirectory .\201-rbac-managedidentity-maps\ -Location eastus`

would run with an error on a conflict of ambiguous names. Just easier to follow.